magicctapipe/lstchain/image/cleaning.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+
+__all__ = ["apply_dynamic_cleaning"]
+
+
+def apply_dynamic_cleaning(image, signal_pixels, threshold, fraction):
```

Comment:
this function is exactly the same as in lstchain 0.10, cant we use it from there?

Reply:
See comment on the lstchain __init__ file.